corehq/apps/domain/views/accounting.py

```diff
@@ -1118,18 +1119,19 @@ def can_change_subscription(self):
         return not is_annual_plan
 
 
-class SelectedEnterprisePlanView(PlanViewBase):
-    template_name = 'domain/selected_enterprise_plan.html'
-    urlname = 'enterprise_request_quote'
+class ContactFormViewBase(PlanViewBase):
+    template_name = 'domain/selected_plan_contact.html'
     step_title = gettext_lazy("Contact Dimagi")
-    edition = SoftwarePlanEdition.ENTERPRISE
+    contact_form = None
+    lead_text = _("Please submit the following form and a member of our sales team will be "
+                 "in touch shortly about your plan.")
```

Comment:
Because class variables get initialized immediately, `lead_text` needs to use `gettext_lazy` (or it will always have the translation from the initial execution context).